commands/user.go

```diff
@@ -131,283 +164,445 @@ func userGroup(b *bot.Bot) {
-						},
-					), "\n"),
-				),
+			var category, parent, children string
+			var list int8
+
+			// Assigns crosspost group and pair variables
+			if group.IsPair {
+				category = locale.Pair
+			} else {
+				category = locale.Group
+				parent = fmt.Sprintf("**%v: **", locale.Parent)
+				children = fmt.Sprintf("**%v: **", locale.Children)
+				list = 1
+			}
+
+			// Assigns field name and description.
+			name := fmt.Sprintf("%v «%v»", category, group.Name)
+			desc := fmt.Sprintf("%v%v\n%v%v",
+				parent,
+				fmt.Sprintf("<#%v> | `%v`", group.Parent, group.Parent),
+				children,
+				strings.Join(arrays.Map(group.Children, func(s string) string {
+					return fmt.Sprintf("<#%v> | `%v`", s, s)
+				}), "\n"),
 			)
+
+			// Appends to groupList.
+			groupList[list] = append(groupList[list], [2]string{name, desc})
+		}
+
+		// Builds crosspost groups into respective fields.
+		for _, list := range groupList {
+			for _, group := range list {
+				eb.AddField(group[0], group[1])
+			}
 		}
 
+		// Returns embed list of crosspost groups.
 		return ctx.ReplyEmbed(eb.Finalize())
 	}
 }
 
-func newgroup(b *bot.Bot) func(ctx *gumi.Ctx) error {
+// newgroup creates a new crosspost group or pair.
+// A crosspost group is given a name and parent channel.
+// A crosspost pair is also given a crosspost channel.
+func newgroup(b *bot.Bot, ispair bool) func(ctx *gumi.Ctx) error {
 	return func(ctx *gumi.Ctx) error {
-		if ctx.Args.Len() < 2 {
+		// Checks if Boe Tea command is correct.
+		//
+		// Crosspost group command suffices with two (2) arguments.
+		// Crosspost pair command suffices with three (3) arguments.
+		if (ispair && ctx.Args.Len() < 3) || ctx.Args.Len() < 2 {
 			return messages.ErrIncorrectCmd(ctx.Command)
 		}
 
+		// Assigns current discord user.
+		// Returns error if no user is found.
 		user, err := b.Store.User(context.Background(), ctx.Event.Author.ID)
 		if err != nil {
 			return err
 		}
 
+		// Assigns name of crosspost group or pair.
 		name := ctx.Args.Get(0).Raw
+		// Assigns name of parent channel.
+		// Returns error if no channel exists.
 		parent := strings.Trim(ctx.Args.Get(1).Raw, "<#>")
 		if _, err := ctx.Session.Channel(parent); err != nil {
 			return messages.ErrChannelNotFound(err, parent)
 		}
 
+		// Assigns a crosspost channel when crosspost group is a pair.
+		// If group is not a pair leave empty.
+		children := []string{}
+		if ispair {
+			// Assigns channelID of crosspost channel.
+			child := strings.Trim(ctx.Args.Get(2).Raw, "<#>")
+
+			// Returns discord channel from channelID.
+			// If no discord channel is found return error.
+			ch, err := ctx.Session.Channel(child)
+			if err != nil {
+				return messages.ErrChannelNotFound(err, child)
+			}
+
+			// Checks if discord channel is a text channel.
+			if ch.Type != discordgo.ChannelTypeGuildText {
+				return messages.ErrIncorrectCmd(ctx.Command)
+			}
+
+			// Checks if crosspost channel is not parent channel.
```

Comment:
Using this comment as an example, but there's a lot of more of them.

These comments are excessive and don't really explain anything, in this particular case `if parent == child` is already obvious what it does just by looking at it.  Some comments can be left, but I'll have to go through them myself.